docs/index.rst

```diff
@@ -77,12 +77,15 @@ and some of the core features of Read the Docs.
 * **Overview of core features**:
   :doc:`Incoming webhooks </webhooks>` |
   :doc:`/custom_domains` |
+  :doc:`/canonical` |
   :doc:`/versions` |
   :doc:`/downloadable-documentation` |
   :doc:`/hosting` |
   :doc:`/server-side-search` |
-  :doc:`/analytics`
-  :doc:`/pull-requests`
+  :doc:`/analytics` |
+  :doc:`/google-analytics` |
```

Comment:
Is this a core feature? Seems a bit overkill here, and like it should be guide content.

Reply:
It is currently a guide, although all too short in my opinion. Shouldn't we merge it into `analytics`, which is also quite short?

Reply:
Ended up merging it, let me know if you think I should revert it